In [12]:
numbers = []
for i in range(0,1000000):
    numbers.append(i+1)

print(f'Min: {min(numbers)} \nMax: {max(numbers)} \nSum: {sum(numbers)}')

Min: 1 
Max: 1000000 
Sum: 500000500000


In [14]:
for i in range(2,31):
    if i%3==0:
        print(i)

3
6
9
12
15
18
21
24
27
30


In [19]:
for i in range(1,10):
    print(i**3)

1
8
27
64
125
216
343
512
729


In [31]:
age=''
while age!= -1:
    age = input('Enter age \nOr write -1 to quit:')
    age = int(age)
    if age== -1:
        break
    elif age<=3:
        print('Free')
    elif age>3 and age<=12:
        print('Ticket price is 1000')
    elif age>12:
        print('Ticket price is 1500')
   

Enter age 
Or write -1 to quit:3
Free
Enter age 
Or write -1 to quit:12
Ticket price is 1000
Enter age 
Or write -1 to quit:15
Ticket price is 1500
Enter age 
Or write -1 to quit:-1


In [74]:
n=input("Enter number: ")
n=int(n)
for i in range(1,n+1,1):
    if n%i==0:
        print(i)
    
        

Enter number: 5
1
5


In [ ]:
summa=[]
n=int(input('Enter number: '))
while n>0:
    a=n%10
    summa.append(a)
    n=n//10
    
print(sum(summa))

6. Напишите такую программу, где вы сначала получите целое число от пользователя. И далее вы суммируете его цифры. Например ввели число 125, то на выходе вы должны получить 1+2+5 = 8. (экспериментируйте)